slack_notification/README.md

```diff
@@ -4,26 +4,50 @@ Hook into platform workflows and post notifications to Slack.
 
 ## Instructions
 
-1. [Enable Incoming Webhooks](https://my.slack.com/services/new/incoming-webhook/) for your Slack instance.
-2. Copy the secret Webhook URL into a file called `secrets.json` and store it in the [private files](https://pantheon.io/docs/articles/sites/private-files/) directory of each environment where you want to trigger Slack notifications. The secret WebHook URL is like a password, which should not be stored in version control.
-
-```shell
-  $> echo '{"slack_url": "https://hooks.slack.com/services/MY/SECRET/URL"}' > secrets.json
-  # Note, you will need to copy the secrets into each environment where you want to trigger Slack notifications.
-  $> `terminus connection:info  --field=sftp_command site.env`
-      Connected to appserver.dev.d1ef01f8-364c-4b91-a8e4-f2a46f14237e.drush.in.
-  sftp> cd files
-  sftp> mkdir private
-  sftp> cd private
-  sftp> put secrets.json
-  sftp> quit
+### Set up the Slack App
+1. [Navigate to api.slack.com/apps](https://api.slack.com/apps) while logged into your Slack workspace.
+1. Click **Create New App**.
+1. Choose **From scratch** in the **Create an app** modal.
+1. Give your app a name (e.g. "Pantheon Deploybot") and select a workspace for your app.
+1. Click **OAuth & Permissions** in the **Features** menu from the left sidebar of your app's configuration screen.
+1. Scroll down to **Bot Token Scopes** under **Scopes** and click the **Add an OAuth Scope** button.
+1. Choose `chat:write` from the dropdown. You may also add other relevant scopes if you plan on extending the Slack notification Quicksilver script's functionality.
+1. Scroll up to **OAuth Tokens** and click the **Install to {your workspace}** button to install the app into your Slack instance.
+1. Authorize ("Allow") the app for your workspace.
+1. Copy the **Bot User OAuth Token** from the **OAuth Tokens** section. We will use [Pantheon Secrets](https://docs.pantheon.io/guides/secrets/overview) to store this token to a secret, bypassing the need for a local file with an API token stored in version control.
+1. You can customize any additional information about your bot, adding an avatar, etc. as you wish.
+
```

Comment:
I needed to do more steps within Slack to get the app into the channel. From the Slack app I added the app. Maybe in previous steps I had only added it to the "workspace" ?

Reply:
Oh right, I needed to invite the app to a channel